test/spec/admin/directives/patient-monthly-list.directive.test.js

```diff
@@ -92,12 +92,12 @@
 			//case undefined
 			isolateScope.locationUuid = undefined;
 			isolateScope.loadPatientList();
-			chai.expect(getPatientsSpy.callCount).to.equal(1);
+			chai.expect(getPatientsSpy.callCount).to.equal(0);
```

Comment:
If you need to change this then change the whole test case. Changing it like this will confuse the next people who will be editing these tests later because the statement says it should be called while you are asserting that it shouldn't be called!